[debug] [2022-11-03T19:35:17.490Z] ----------------------------------------------------------------------
[debug] [2022-11-03T19:35:17.492Z] Command:       /usr/bin/node /usr/bin/firebase deploy
[debug] [2022-11-03T19:35:17.493Z] CLI Version:   11.16.0
[debug] [2022-11-03T19:35:17.493Z] Platform:      linux
[debug] [2022-11-03T19:35:17.493Z] Node Version:  v18.11.0
[debug] [2022-11-03T19:35:17.494Z] Time:          Thu Nov 03 2022 15:35:17 GMT-0400 (Eastern Daylight Time)
[debug] [2022-11-03T19:35:17.494Z] ----------------------------------------------------------------------
[debug] 
[debug] [2022-11-03T19:35:17.570Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2022-11-03T19:35:17.571Z] > authorizing via signed-in user (jtb0032@mix.wvu.edu)
[debug] [2022-11-03T19:35:17.571Z] [iam] checking project human-benchmark-app for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2022-11-03T19:35:17.573Z] >>> [apiv2][query] POST https://cloudresourcemanager.googleapis.com/v1/projects/human-benchmark-app:testIamPermissions [none]
[debug] [2022-11-03T19:35:17.574Z] >>> [apiv2][(partial)header] POST https://cloudresourcemanager.googleapis.com/v1/projects/human-benchmark-app:testIamPermissions x-goog-quota-user=projects/human-benchmark-app
[debug] [2022-11-03T19:35:17.574Z] >>> [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/human-benchmark-app:testIamPermissions {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2022-11-03T19:35:17.941Z] <<< [apiv2][status] POST https://cloudresourcemanager.googleapis.com/v1/projects/human-benchmark-app:testIamPermissions 401
[debug] [2022-11-03T19:35:17.941Z] <<< [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/human-benchmark-app:testIamPermissions {"error":{"code":401,"message":"Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.","status":"UNAUTHENTICATED","details":[{"@type":"type.googleapis.com/google.rpc.ErrorInfo","reason":"ACCESS_TOKEN_TYPE_UNSUPPORTED","metadata":{"service":"cloudresourcemanager.googleapis.com","method":"google.cloudresourcemanager.v1.Projects.TestIamPermissions"}}]}}
[debug] [2022-11-03T19:35:17.942Z] [iam] error while checking permissions, command may fail: FirebaseError: HTTP Error: 401, Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.
[debug] [2022-11-03T19:35:17.944Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/human-benchmark-app [none]
[debug] [2022-11-03T19:35:18.176Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects/human-benchmark-app 401
[debug] [2022-11-03T19:35:18.176Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/human-benchmark-app {"error":{"code":401,"message":"Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.","status":"UNAUTHENTICATED"}}
[debug] [2022-11-03T19:35:18.176Z] HTTP Error: 401, Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.
[debug] [2022-11-03T19:35:18.341Z] FirebaseError: HTTP Error: 401, Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.
    at responseToError (/usr/lib/node_modules/firebase-tools/lib/responseToError.js:49:12)
    at RetryOperation._fn (/usr/lib/node_modules/firebase-tools/lib/apiv2.js:288:77)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[error] 
[error] Error: Failed to get Firebase project human-benchmark-app. Please make sure the project exists and your account has permission to access it.
